org.zulip.Zulip.yaml

```diff
@@ -48,7 +48,9 @@ modules:
         sha256: 62472bc97fd702da1d526b3e9a605884679f7e0cd8a067b6b6369a2cbee2bd11
         dest-filename: SHASUMS256.txt-8.0.0
       - type: file
-        path: org.zulip.Zulip.appdata.xml
+        path: org.zulip.Zulip.desktop
+      - type: file
+        path: org.zulip.Zulip.metainfo.xml
```

Comment:
Why is this shipped as a generic metainfo.xml instead of an appdata.xml? The latter seemed to be correct: https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html